pkg/codegen/schema/schema.go

```diff
@@ -339,6 +348,8 @@ type Function struct {
 	DeprecationMessage string
 	// Language specifies additional language-specific data about the function.
 	Language map[string]interface{}
+	// IsMethod indicates whether the function is a method of a resource.
+	IsMethod bool
```

Comment:
This will be useful for the forthcoming codegen changes, to avoid emitting a top-level function if the function is being used as a method.